### ISSUE TITLE
google_chrome.cache error

### ISSUE THREAD

**Issue by Saibamen** [bleachbit/bleachbit#938]
```
Błąd: google_chrome.cache: Command to clean JSON file, path=C:\Users\UserXXX\AppData\Local\Google\Chrome\User Data\Default\Preferences, address=dns_prefetching/host_referral_list 
Traceback (most recent call last):
  File "C:\projects\bleachbit\bleachbit\Worker.py", line 87, in execute
  File "C:\projects\bleachbit\bleachbit\Command.py", line 244, in execute
  File "C:\projects\bleachbit\bleachbit\FileUtilities.py", line 263, in clean_json
  File "C:\Python34\lib\json\__init__.py", line 265, in load
  File "C:\Python34\lib\encodings\cp1250.py", line 23, in decode
UnicodeDecodeError: 'charmap' codec can't decode byte 0x98 in position 28203: character maps to <undefined>
Błąd: google_chrome.cache: Command to clean JSON file, path=C:\Users\UserXXX\AppData\Local\Google\Chrome\User Data\Default\Preferences, address=dns_prefetching/startup_list 
Traceback (most recent call last):
  File "C:\projects\bleachbit\bleachbit\Worker.py", line 87, in execute
  File "C:\projects\bleachbit\bleachbit\Command.py", line 244, in execute
  File "C:\projects\bleachbit\bleachbit\FileUtilities.py", line 263, in clean_json
  File "C:\Python34\lib\json\__init__.py", line 265, in load
  File "C:\Python34\lib\encodings\cp1250.py", line 23, in decode
UnicodeDecodeError: 'charmap' codec can't decode byte 0x98 in position 28203: character maps to <undefined>
Błąd: google_chrome.cache: Command to clean JSON file, path=C:\Users\UserXXX\AppData\Local\Google\Chrome\User Data\Default\Preferences, address=net/http_server_properties/servers 
Traceback (most recent call last):
  File "C:\projects\bleachbit\bleachbit\Worker.py", line 87, in execute
  File "C:\projects\bleachbit\bleachbit\Command.py", line 244, in execute
  File "C:\projects\bleachbit\bleachbit\FileUtilities.py", line 263, in clean_json
  File "C:\Python34\lib\json\__init__.py", line 265, in load
  File "C:\Python34\lib\encodings\cp1250.py", line 23, in decode
UnicodeDecodeError: 'charmap' codec can't decode byte 0x98 in position 28203: character maps to <undefined>
```

BleachBit version 4.0.0
Google Chrome 83.0.4103.97
Windows Server 2019 Standard

**Comment by az0** [comment]
This looks like the character encoding issue like https://github.com/bleachbit/bleachbit/issues/874

**Comment by az0** [comment]
@Saibamen 
When you get a chance, would you please download version 4.0.1.1660 or newer from https://ci.bleachbit.org/ and test it? It should be there in ~15 minutes. Afterwards, you should not have have the UnicodeDecodeError when cleaning Google Chrome

**Comment by Saibamen** [comment]
Fixed - 4.0.1.1662 - setup version

**Comment by az0** [comment]
thank you for checking, @Saibamen
